File: Manual 02/Question07.cpp
#include <iostream>
using namespace std;

int main()
{
    int n, f, f1 = -1, f2 = 1;

    cout << "Enter the Number of Terms: ";
    cin >> n;
    cout << "The Series is as follows: " << endl;
    while (n > 0)
    {
        f = f1 + f2;
        f1 = f2;
        f2 = f;
        cout << f << ", ";
        n--;
    }
}

File: Python/Manual 02/Question07.py
a = 0
b = 1

n = int(input("Enter Number of Terms: "))

for i in range(n + 1):
    if i == 0:
        print(0)
    elif i == 1:
        print(1)
    else:
        c = a + b
        a = b
        b = c
        print(str(c))
        n -= 1
